week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -57,11 +76,32 @@ function exerciseTwo(shopping) {
 
     The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
 **/
-function exerciseThree(books) {
+ function exerciseThree(books) {
+  let content = document.querySelector("#content");
+  let bookList = document.createElement("ul");
+   bookList.style.display = "flex"
+   bookList.style.listStyleType = "none"
+   books.forEach(function(book){
+     let listItem = document.createElement("li");
+     let paragraph = document.createElement("p");
+     paragraph.textContent = `${book.title}  - ${book.author}`
+     listItem.style.style.backgroundColor = book.alreadyRead ? "green"  : "red"
```

Comment:
![image](https://user-images.githubusercontent.com/5181870/79276715-6b66fc80-7ea0-11ea-9d1e-63bbe4707cf0.png)

I'm seeing this error when I check this code. Could you fix this?